tests/console/http_srv.pm

```diff
@@ -11,26 +11,46 @@
 # Summary: Simple apache server test
 # Maintainer: Oliver Kurz <okurz@suse.de>
 
+package http_srv;
+
+use base Exporter;
+use Exporter;
+
 use strict;
 use warnings;
 use base "consoletest";
 use testapi;
 use utils;
+our @EXPORT = qw(check_apache_install
+  enable_apache_config
+  start_apache_service
+  check_apache_service
+  check_apache_funtion);
 
-sub run {
-    select_console 'root-console';
-    # Log space available before installation, see poo#19834
-    script_run("df -h > /dev/$serialdev", 0);
-    # Install apache2
+# install package
+sub check_apache_install {
     zypper_call("in apache2");
+}
+
+# configure apache
+sub enable_apache_config {
```

Comment:
`enable_apache_config` is confusing for me . you enabling service here , also it will look more clear together with `start_apache_service` and `check_apache_service`